src/lib/api/api.go

```diff
@@ -172,45 +172,41 @@ func (a *API) RoutesSummary(w http.ResponseWriter, _ *http.Request) {
 func (a *API) Query(w http.ResponseWriter, r *http.Request) {
 	var (
 		startTimestamp, endTimestamp int64
-		err                          error
 	)
 	timeSeriesPath := r.URL.Query().Get("timeSeriesPath")
-
 	startTimestampStr := r.URL.Query().Get("startTimestamp")
 	if startTimestampStr == "" {
 		startTimestamp = int64(math.MaxInt64)
 	} else {
-		startTimestamp, err = strconv.ParseInt(startTimestampStr, 10, 64)
+		t, err := time.Parse(time.RFC3339, startTimestampStr)
```

Comment:
Can we make changes in UI to return in the format of [`http.TimeFormat`?](https://golang.org/pkg/net/http/#pkg-constants) That is some expected standard that we should do, but nothing hard rule here.

Reply:
Sure will do that

Reply:
@Harkishen-Singh unlike iso format http.TimeFormat dont have inbuilt pattern conversion in both moment js as well as golang so do you think we should convert it to that format by manually writting the pattern we want it to be converted to? or shall we continue with iso string only